Packs/Core/Playbooks/playbook-Credential_Dumping_using_a_known_tool.yml

```diff
@@ -0,0 +1,628 @@
+              simple: AGENT_ISOLATED
+          ignorecase: true
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 10,
+          "y": 10
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 2
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "32":
+    id: "32"
+    taskid: e779dfc7-1f39-4ea1-8395-693901916095
+    type: regular
+    task:
+      id: e779dfc7-1f39-4ea1-8395-693901916095
+      version: -1
+      name: Terminate Causality (CGO)
+      description: Terminate a process by it's causality ID.
+      script: '|||core-terminate-causality'
+      type: regular
+      iscommand: true
+      brand: ""
+    nexttasks:
+      '#error#':
+      - "33"
+      '#none#':
+      - "28"
+    scriptarguments:
+      agent_id:
+        simple: ${alert.agentid}
+      causality_id:
+        simple: ${alert.cid}
+      timeout_in_seconds:
+        simple: "180"
+    separatecontext: false
+    continueonerror: true
+    continueonerrortype: errorPath
+    view: |-
+      {
+        "position": {
+          "x": -240,
+          "y": -650
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "33":
+    id: "33"
+    taskid: f82b7572-1a45-4d6a-84b5-b6802dcc44af
+    type: regular
+    task:
+      id: f82b7572-1a45-4d6a-84b5-b6802dcc44af
+      version: -1
+      name: Terminate Causality Process Manually
+      description: "Dear Analyst,\n\nDuring the remediation process, the playbook failed to terminate the causality process ${alert.cgoname} \n\nPlease manually terminate the process if possible. \nNote that the next remediation step, if possible, will be endpoint isolation."
+      type: regular
+      iscommand: false
+      brand: ""
+    nexttasks:
+      '#none#':
+      - "28"
+    separatecontext: false
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": -430,
+          "y": -460
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "36":
+    id: "36"
+    taskid: ca7ef243-bf2e-4de4-8e0c-d44f7703cd0f
+    type: regular
+    task:
+      id: ca7ef243-bf2e-4de4-8e0c-d44f7703cd0f
+      version: -1
+      name: Manual remediation actions for a server or a disconnected endpoint
+      description: "Dear Analyst,\n\nPlease note that during the remediation process, the playbook will not isolate the following host: ${alert.hostname} \n\nThis is due to one of the following reasons:\n- The device disconnected.\n- The device has been identified as a server.\n\nPlease take manual action to contain the attack and prevent the attacker from executing lateral movement before closing this alert."
```

Comment:
```suggestion
      description: "Dear Analyst,\n\nPlease note that during the remediation process, the playbook didn't isolate the following host: ${alert.hostname} \n\nThis is due to one of the following reasons:\n- The device disconnected.\n- The device has been identified as a server.\n\nPlease take manual action to contain the attack and prevent the attacker from spreading in your network."
```

Reply:
Agreed. I only changed the first part.

Dear Analyst,

Please note that during the remediation process, the playbook didn't isolate the following host: ${alert.hostname} 

This is due to one of the following reasons:
- The device disconnected.
- The device has been identified as a server.

Please take manual action to contain the attack and prevent the attacker from executing lateral movement before closing this alert.